config/features.yml

```diff
@@ -1166,6 +1166,9 @@ features:
   va_burial_v2:
     actor_type: user
     description: Allows us to toggle between 21-P530 and 21-P530V2
+  veteran_onboarding_beta_flow:
+    actor_type: user
+    description: Conditionally display the new veteran onboarding flow to user
```

Comment:
I actually added the feature toggle in [16573](https://github.com/department-of-veterans-affairs/vets-api/pull/16573) but we are experiencing issues with the `master` branch (which has been merged into that branch).  Once 16573 has been merged into `master` and the other issues are resolved, this change will not be present in the current PR.